data/fedora-server.css

```diff
@@ -0,0 +1,63 @@
+@define-color fedora #2f4265;
+
+/* logo and sidebar classes for Fedora */
+
+/* The sidebar consists of three parts: a background, a logo, and a product logo,
+ * rendered in that order. The product logo is empty by default and is intended
+ * to be overridden by a stylesheet in product.img.
+ */
+.logo-sidebar {
+    background-image: url('/usr/share/anaconda/pixmaps/server/sidebar-bg.png');
+    background-color: @fedora;
+    background-repeat: no-repeat;
+}
+
+/* Add a logo to the sidebar */
+.logo {
+    background-image: url('/usr/share/anaconda/pixmaps/server/sidebar-logo.png');
+    background-position: 50% 20px;
+    background-repeat: no-repeat;
+    background-color: transparent;
+}
+
+/* This is a placeholder to be filled by a product-specific logo. */
+.product-logo {
+    background-image: none;
+    background-color: transparent;
+}
+
+AnacondaSpokeWindow #nav-box {
+    background-color: @fedora;
+    background-image: url('/usr/share/anaconda/pixmaps/server/topbar-bg.png');
+    background-repeat: no-repeat;
+    color: white;
+}
+
+/* Remove the box-shadow from buttons in the nav-box because it adds a white stripe
+ * below the buttons and makes them look dumb */
+AnacondaSpokeWindow #nav-box GtkButton {
+    box-shadow: none;
+}
+
+
+/* Change the sidebar background color */
+.logo-sidebar {
+  background-color: #006eb4;
+}
+
+AnacondaSpokeWindow #nav-box {
+  background-color: #006eb4;
+}
+
+/* Move the Fedora logo to the bottom */
+.logo {
+    background-position: 50% 90%;
+}
+
+/* Add the product logo at the top */
+.product-logo {
+    background-image: url('/usr/share/anaconda/pixmaps/server/sidebar-logo_flavor.png');
```

Comment:
symlink to fedora-logos created by productimg package

Reply:
This is not a symlink created by productimg package. This is the literal path to the content in fedora-logos.

The productimg package *does* create symlinks right now, but this was a mistake and I fixed it in this PR. I'm not sure why I did that originally, but it was incorrect. I suspect it was because the CSS was written by someone else (modified from the official one) and I opted to move the images into the locations expected by the CSS instead of fixing the CSS.